Lib/Inc/IComChannel.h

```diff
@@ -27,7 +27,7 @@ class IComChannel
 
     virtual bool isCommandAvailable() = 0;
     virtual std::string getCommand() = 0;
-    virtual void send(std::string message) = 0;
+    virtual void send(const std::string &message) = 0;
```

Comment:
I changed this to const std::string & because we are not changing the message and it is passed by reference to avoid copying it.